crates/bevy_text/src/text.rs

```diff
@@ -29,8 +29,6 @@ impl Default for CosmicBuffer {
 #[derive(Component, Debug, Clone, Default, Reflect)]
 #[reflect(Component, Default)]
 pub struct Text {
```

Comment:
```suggestion
pub struct TextNode {
```

Reply:
This suggestion seems confusing because the component is also used in `Text2dBundle`, and we don't use this naming convention in any other UI bundles.

Reply:
What do we think of `TextLayout`?

Reply:
I'll go with `TextLayout`, makes the most sense to me.